.github/workflows/multibuild.yml

```diff
@@ -20,6 +20,11 @@ jobs:
         PLAT: [i686, x86_64]
         INTERFACE64: ['', '1']
         MB_ML_VER: ['', 2010, 2014]
+        include:
+          - os: macos-latest
+            PLAT: arm64
+            INTERFACE64: ''
+            platform: [x64]
```

Comment:
I am not sure we need `platform` at all in this file, it might be some cruft left over from the transition to github actions?

Reply:
Maybe add a comment to explain that this entry of the build matrix is using cross-compilation to build an macos/arm64 on a macos/x86_64 host (typically on github actions that does not provide native macos/arm64  runners at this time).